### PR TITLE
Restore edit on github link for guidelines

### LINKED ISSUE #80: Restore "edit" links
Before the restyle, we had an "edit" link on all articles & documents, pointing to Github's online edition feature, to allow users to easily fix typos or propose minor changes.

Having this edit link was decided as a group (by the 'roo tech leads) so we'd probably want it back!

### PR DESCRIPTION
**Pull request by kapowaz**
Fixes #80.

### CODE REVIEW

**Comment by kapowaz** [comment]
img

File: _layouts/guidelines.html [hunk 1]
```diff
@@ -4,7 +4,10 @@
 <article class="guidelines">
   <header>
     <h1>{{ page.title }}</h1>
-    <p class="tagline">{{ page.subtitle }}</p>
+    <p class="tagline">
+      {{ page.subtitle }}
+      <a href="{{site.github.repository_url}}/blob/gh-pages/{{page.path}}" class="edit" title="Edit these guidelines on Github" target="_blank">Edit</a>
```
Comment (mezis) [inline _layouts/guidelines.html]: That URL seems wrong — missing `/edit/`.
For instance, this links directly to an edit screen:

https://github.com/deliveroo/deliveroo.engineering/edit/gh-pages/README.md

Comment (kapowaz) [inline _layouts/guidelines.html]: Oops! Not sure how that happened, I’ll fix that now 😁

**Comment by mezis** [comment]
Looks rad :+1:

**Pull request merged**